index.html

```diff
@@ -87,5 +95,52 @@
         hidePreloader();
         });
         </script>
-    </body>
+
+    <!---------------------------------->
+
+
+    <!-----------Main Body--------->
+    <div id="video">
+      <video autoplay loop>
+        <source src="assets/video/homepage/doc2pen video.m4v" type="video/mp4">
+      </video>
+    </div>
+
+    <div class="title-text">
+        <div id="header">
+          <p id="doc2pen">Doc2Pen</p>
+          <p id="tagline">A Student's Life Saviour</p>
+        </div>
+
+        <div class="logo">
+          <img id="img-logo" src="/assets/favicon/logo.png">
+        </div>
+    </div>
+
+    <div class="what">
+        <p id="what-text">What is DOC2PEN?</p>
+        <img id="what-image" src="assets/images/To_do_list.svg">
+        <p id="what-ans">DOC2PEN is a student's saviour to submit the assignments by just typing them out and getting it handwritten on the go! It simply converts image to handwritten document and and also allow us to download documents easily.</p>
+    </div>
+
+    <div class="features-block">
+      <p id="key-title">Key Features</p>
+      <ul id="row">
+        <li id="column">Convert Typed Document to Handwritten</li>
+        <li id="column">Custom Fonts Available</li>
+        <li id="column">Download result in various formats</li>
+        <li id="column">Open Source</li>
+      </ul>
+    </div>
+
+    <div class="contribute-block">
+      <p id="contribute-heading">Think You can <br><span id="contribute">Contribute?</span></p>
```

Comment:
This should be in the contact page. Home page is only meant to serve as a pictorial elevator pitch